usecases/connect-autonomous-database/src/main/java/com/oracle/cloud/world/demo/repository/CountryRepository.java

```diff
@@ -0,0 +1,16 @@
+package com.oracle.cloud.world.demo.repository;
+
+import javax.transaction.Transactional;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+import org.springframework.transaction.annotation.EnableTransactionManagement;
+
+import com.oracle.cloud.world.demo.model.Country;
```

Comment:
Suggested to remove Cloud World reference (generalize)...

Reply:
Renamed